lib/eex/lib/eex/engine.ex

```diff
@@ -133,11 +140,23 @@ defmodule EEx.Engine do
     end
   end
 
+  def handle_expr(_buffer, "/", _expr) do
+    not_implemented!("/")
+  end
+
+  def handle_expr(_buffer, "|", _expr) do
+    not_implemented!("|")
+  end
+
   def handle_expr(buffer, "", expr) do
     quote do
       tmp2 = unquote(buffer)
       unquote(expr)
       tmp2
     end
   end
+
+  defp not_implemented!(marker) do
```

Comment:
I think we should call this function `raise_marker_not_implemented/1` as we usually reserve the `!` at the end of function names for when there's a function that doesn't raise and a variant of that function that raises.